textstat/textstat.py

```diff
@@ -605,6 +605,48 @@ def crawford(self, text):
 
         return legacy_round(craw_years, 1)
 
+    @lru_cache(maxsize=128)
+    def longwordcount(self, text):
```

Comment:
```suggestion
    def long_word_count(self, text):
```

Reply:
hi @alxwrd! 
Thanks for taking a look and reviewing the PR. I totally agree with you on the grouping of the counting functions and the index functions. However, I thought that you are following a convention where the used utility functions are directly above the index function like it is e.g. done [here](https://github.com/shivam5992/textstat/blob/e8e33a2f5de2769f2097e2af405cfd4ed379585a/textstat/textstat.py#L247) with `polysyllabcount()` and `smog_index()`. 

So, as far as I see it there are two options to structure the class that I could also directly implement:
- move all counting functions to the very top of the class
- refactor all counting functions to a utility script 

The second option might be cleaner overall since it declutters the main script and the class in general. Overall most of these little functions could also be static and hence "live" somewhere else. What are your thoughts about this?

Reply:
hi, sorry for the delay. I think all the counting function for now should just live together at the top of the class, but this is probably an exercise during some refactoring later.